src/Sarif.FunctionalTests/v2/ConverterTestData/PREfast/Expected-022.xml.sarif

```diff
@@ -1,6 +1,6 @@
 {
-  "$schema": "http://json.schemastore.org/sarif-2.0.0-csd.2.beta.2018-09-26",
-  "version": "2.0.0-csd.2.beta.2018-09-26",
+  "$schema": "http://json.schemastore.org/sarif-2.0.0-csd.2.beta.2018-11-28",
```

Comment:
>$schema [](start = 3, length = 7)

A small # of test files had some inconsistencies that would have required manual updates to resolve. Instead, I've simply converted them to the current version of SARIF. We have plenty of test content that continues to be transformed (in fact, the bulk of our test content is still pre-release v2). It is impressive that our transformer continues to allow all our testing to function.